ddp-workspace/projects/toolkit/src/lib/components/dialogs/sessionWillExpire.component.ts

```diff
@@ -77,7 +83,13 @@ export class SessionWillExpireComponent implements OnInit, OnDestroy {
     public renewSession(): void {
         // Blocks UI to prevent interference in the session renewing process
         this.isRenewing = true;
-        this.auth0.auth0RenewToken();
+        this.auth0.auth0RenewToken().pipe(
+            take(1),
+            finalize(() => this.isRenewing = false)
+        ).subscribe(
+            () => true,
```

Comment:
Can we make empty function instead of true?
```suggestion
            () => { },
```